authpass/lib/ui/screens/entry_details.dart

```diff
@@ -1548,40 +1549,57 @@ class ObscuredEntryFieldEditor extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Stack(
+      alignment: Alignment.centerRight,
       children: [
-        InputDecorator(
-          decoration: InputDecoration(
-            prefixIcon:
-                commonField?.icon == null ? null : Icon(commonField.icon),
-            labelText: commonField?.displayName ?? fieldKey.key,
-            filled: true,
-          ),
-          child: const Text(
-            '*****************',
-            style: TextStyle(color: Colors.white),
-          ),
-        ),
-        Positioned.fill(
-          child: ClipRect(
-            child: BackdropFilter(
-              filter: ui.ImageFilter.blur(
-                sigmaX: 0.5,
-                sigmaY: 0.5,
+        Stack(
+          children: [
+            InputDecorator(
+              decoration: InputDecoration(
+                prefixIcon:
+                    commonField?.icon == null ? null : Icon(commonField.icon),
+                labelText: commonField?.displayName ?? fieldKey.key,
+                filled: true,
+              ),
+              child: const Text(
+                '*****************',
+                style: TextStyle(color: Colors.white),
               ),
-              child: LinkButton(
-                child: Container(
-                  alignment: Alignment.bottomCenter,
-                  padding: const EdgeInsets.only(left: 12.0 + 24.0, bottom: 16),
-                  child: const Text(
-                    'Protected field. Click to reveal.',
-                    style: TextStyle(
-                        shadows: [Shadow(color: Colors.white, blurRadius: 5)]),
+            ),
+            Positioned.fill(
+              child: ClipRect(
+                child: BackdropFilter(
+                  filter: ui.ImageFilter.blur(
+                    sigmaX: 0.5,
+                    sigmaY: 0.5,
+                  ),
+                  child: LinkButton(
+                    child: Container(
+                      alignment: Alignment.bottomCenter,
+                      padding: const EdgeInsets.only(left: 12.0 + 24.0, bottom: 16, right: 12),
+                      child: const Text(
+                        'Protected field. Click to reveal.',
+                        style: TextStyle(
+                            shadows: [Shadow(color: Colors.white, blurRadius: 5)]),
+                      ),
+                    ),
+                    onPressed: onPressed,
                   ),
                 ),
-                onPressed: onPressed,
               ),
             ),
-          ),
+          ],
+        ), 
+        IconButton(
+          icon: Icon(FontAwesomeIcons.eye),
```

Comment:
This should be `const` - is your IDE not showing the warning? 🤔️

<img width="452" alt="Screen Shot 2020-09-29 at 13 25 38" src="https://user-images.githubusercontent.com/313066/94552495-45b78f00-0257-11eb-92e0-75b6e79ca3a8.png">